ContosoCrafts.WebSite/Models/User.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Security.Cryptography;
+using System.Text;
+using System.Text.Json;
+
+
+namespace ContosoCrafts.WebSite.Models
+{
+	public class User
+	{
+        public string UserId { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public DateTime JoinedAt { get; set; }
+        public string Username { get; set; }
+        public string PasswordHash { get; set; }
+        public string ProfilePictureUrl { get; set; }
```

Comment:
Have we decided how we want to identify user's profile pictures?

Reply:
I'm not sure I understand entirely what you are asking but I don't think we have. We can discuss it in the group.

Reply:
I think we should discuss more about what user data should look like and where the data will go since I think prof mentioned something about databases in class